project.clj

```diff
@@ -51,34 +54,37 @@
                    :sql-store :sql-store
                    :es-store :es-store
                    :disabled :disabled
-                   :es-producer #(or (:es-producer %)
-                                     (:es-producer-filtered-alias %)
-                                     (:es-producer-aliased-index %))
-
-                   :default #(not (:disabled %))
-                   :integration #(or (:es-store %)
-                                     (:integration %)
-                                     (:es-producer %)
-                                     (:es-producer-filtered-alias %)
-                                     (:es-producer-aliased-index %))
+                   :es-producer #(#{:es-producer
+                                    :es-producer-filtered-alias
+                                    :es-producer-aliased-index}
+                                  %)
+                   :default #(not= :disabled %)
+                   :integration #(#{:es-store
+                                    :integration
+                                    :es-producer
+                                    :es-producer-filtered-alias
+                                    :es-producer-aliased-index}
+                                  %)
```

Comment:
The sets in these functions are already functions, and don't need to be wrapped in an anonymous function. Just leave them as sets.
e.g.
`:es-producer #{:es-producer
                :es-producer-filtered-alias
                :es-producer-aliased-index}`